app/models/user.rb

```diff
@@ -32,8 +31,8 @@ class User < ApplicationRecord
   delegate   :super_admin_user?, :admin_user?, :self_service?, :limited_self_service?,
              :to => :miq_user_role, :allow_nil => true
 
-  validates_presence_of   :name, :userid, :region
-  validates_uniqueness_of :userid, :scope => :region
+  validates_presence_of   :name, :userid
+  validates :userid, :uniqueness => {:conditions => -> { in_my_region } }
```

Comment:
I think we didn't. At least not in this pr.

I think we would make a some unhappy: John Doe, Jan Novak, Mohammed Abdullah or Zhang Wei.